api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TPTInitiationCohortQueries.java

```diff
@@ -309,7 +217,8 @@ public CohortDefinition getPatientsWithUltimaProfilaxia3hp() {
             + "       AND e.location_id = :location "
             + "       AND e.encounter_type = ${53} "
             + "       AND ( o.concept_id = ${23985} AND o.value_coded = ${23954} ) "
-            + "       AND ( o2.concept_id = ${6128} AND o2.value_datetime BETWEEN :startDate AND :endDate ) ";
+            + "       AND ( o2.concept_id = ${6128} "
```

Comment:
Check the requirements.. should use new structure to check the 3HP Start Date. Valude_coded = Iniciar (id= 1256) for concept_id=165308 ObsDateTime for concept id = 165308

Reply:
Applied according To requirement